server/models/userModal.js

```diff
@@ -28,7 +28,11 @@ const userSchema = new Schema(
       trim: true,
       unique: true,
     },
-    password: { type: String, required: [true, 'Email or Password Incorrect'] },
+    password: {
+      type: String,
+      required: [true, 'Email or Password Incorrect'],
+      select: false,
```

Comment:
This is the change if you reviewing.

Reply:
Yup, that can be done as well. We'll go ahead with this. Thanks